src/main/resources/assets/patchouli/lang/de_DE.lang

```diff
@@ -0,0 +1,93 @@
+# Item Names
+item.patchouli:guide_book.name=Reiseführer
+
+# Testing
+item.patchouli:test_book_1.name=Testbuch 1
+item.patchouli:test_book_1.landing=Nicht so toll wie eine Einführung, es ist aber trotzdem präsent.
+
+item.patchouli:test_book_2.name=Testbuch 2
+item.patchouli:test_book_2.landing=Einführungstext! Es unterstützt auch $(2)Farben$() und $(4)solche $(bold)Sachen$()!
+
+item.patchouli:test_completion.name=Fortschritts-Testbuch
+item.patchouli:test_completion.landing=Das Buch ändert die Farben, wenn du Einträge freischaltest!
+
+item.patchouli:intro_book.name=Patchouli
+item.patchouli:intro_book.subtitle=Dokumentation für jedermann
+item.patchouli:intro_book.landing=Mit $(item)Patchouli$() machst du es dir einfach, fortschrittliche freischaltbare $(thing)Bücher$(), zu lesen für Modifikationen und Modifikationenpakete!
+
+
+#Subtitles
+patchouli.subtitle.book_open=Buch, das geöffnet wird
+patchouli.subtitle.book_flip=Seitenumschläge
+
+
+
+# Gui Strings
+patchouli.gui.lexicon.landing_info=(Platzhalter für den Einführungstext, bestimme dein eigenes in der json-Datei deines Buches!)
+patchouli.gui.lexicon.progress_meter=Fortschritt freischalten:
+patchouli.gui.lexicon.progress_tooltip=%d/%d Freigeschaltet
+patchouli.gui.lexicon.progress_tooltip.secret1=Plus 1 Geheimnis
+patchouli.gui.lexicon.progress_tooltip.secret=Plus %d Geheimnisse
+patchouli.gui.lexicon.progress_tooltip.info=Mache Fortschritte um noch mehr freizuschalten!
+patchouli.gui.lexicon.categories=Kategorien
+patchouli.gui.lexicon.chapters=Kapitel
+patchouli.gui.lexicon.editor=Buch-Bearbeitungsprogramm
+patchouli.gui.lexicon.editor.mock_header=Kopfzeile der Nachbildung
+patchouli.gui.lexicon.editor.info=Dies ist ein Bearbeitungsprogramm für Patchouli's-Bucheinträge. Damit ist gemeint, dass es für die Entwicklung oder Übersetzung genutzt wird. Es gibt dafür kein Nutzen wenn du ein Spieler bist.$(br2)Bitte achte auf dem $(l:https://github.com/Vazkii/Patchouli/wiki/Text-Formatting-101)Patchouli Wiki$() für nützliche Steuercodes.
+patchouli.gui.lexicon.shift_for_recipe=Umschalttaste-klicken für Rezept
+patchouli.gui.lexicon.index=Eintragsverzeichnis
+patchouli.gui.lexicon.index.info=Ein Verzeichnis von jedem Kapitel ist Verfügbar in diesem Buch.$(br2)Du kannst den Verzeichnis durchsuchen oder irgendwelche andere Kategorie, ganz einfach durch das Eintippen der Anfrage. Die Suchleiste wird dann erscheinen um dich zu unterstützen.
```

Comment:
This translation is just completely wrong. Please read the original correctly when translating
```suggestion
patchouli.gui.lexicon.index.info=Ein Verzeichnis aller in diesem Buch zu findenden Kapitel.$(br2)Du kannst das Verzeichnis sowie jegliche andere Kategorie durchsuchen, indem zu anfängst, Begriffe einzutippen. Eine Suchleiste wird dann erscheinen, um dir bei der Suche zu helfen.
```